x-pack/plugins/siem/public/management/pages/policy/view/policy_list.tsx

```diff
@@ -89,63 +95,110 @@ export const PolicyList = React.memo(() => {
   const columns: Array<EuiTableFieldDataColumnType<Immutable<PolicyData>>> = useMemo(
     () => [
       {
-        field: 'name',
+        field: '',
         name: i18n.translate('xpack.siem.endpoint.policyList.nameField', {
           defaultMessage: 'Policy Name',
         }),
         // eslint-disable-next-line react/display-name
-        render: (value: string, item: Immutable<PolicyData>) => {
+        render: (item: Immutable<PolicyData>) => {
           const routePath = getManagementUrl({
             name: 'policyDetails',
             policyId: item.id,
             excludePrefix: true,
           });
           const routeUrl = getManagementUrl({ name: 'policyDetails', policyId: item.id });
-          return <PolicyLink name={value} route={routePath} href={routeUrl} />;
+          return (
+            <>
+              <EuiFlexItem grow={false}>
+                <PolicyLink name={item.name} route={routePath} href={routeUrl} />
+              </EuiFlexItem>
+              <EuiFlexItem>
+                <EuiText
+                  color="subdued"
+                  size="xs"
+                  style={{ marginLeft: '6px', whiteSpace: 'nowrap' }}
+                >
+                  <FormattedMessage
+                    id="xpack.siem.endpoint.policyList.revision"
+                    defaultMessage="rev. {revNumber}"
+                    values={{ revNumber: item.revision }}
+                  />
+                </EuiText>
+              </EuiFlexItem>
+            </>
+          );
         },
         truncateText: true,
       },
       {
-        field: 'revision',
-        name: i18n.translate('xpack.siem.endpoint.policyList.revisionField', {
-          defaultMessage: 'Revision',
+        field: 'created_by',
+        name: i18n.translate('xpack.siem.endpoint.policyList.createdBy', {
+          defaultMessage: 'Created By',
         }),
-        dataType: 'number',
+        truncateText: true,
       },
       {
-        field: 'package',
-        name: i18n.translate('xpack.siem.endpoint.policyList.versionField', {
-          defaultMessage: 'Version',
+        field: 'created_at',
```

Comment:
:grey_question: maybe "created_on"?

Reply:
Similar to my previous response, this refers to the name of the field in state, resulting from the API response.